package.json

```
@@ -22,7 +23,11 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint src/**/*.{js,jsx,ts,tsx,json}",
+    "lint:fix": "eslint --fix src/**/*.{js,jsx,ts,tsx,json}",
```

It can be simplified as we won't use js, jsx files and I'm not sure if we need to run eslint for json...
```suggestion
    "lint": "eslint . --ext .{ts,tsx}",
    "lint:fix": "eslint --fix . --ext .{ts,tsx}",
```